tests/openassetio/test/manager/resources/fixtures_cliPass.py

```diff
@@ -14,13 +14,13 @@
 #   limitations under the License.
 #
 """
-Manager test harness test case fixtures for managerValidator test
+Manager test harness test case fixtures for manager test
```

Comment:
[consider][minor] A bit confusing now it says "manager test", perhaps "manager test harness tests"

Reply:
Reworded [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/184/commits/72aebad69a5ebe77d2a97ab61116d5402d7749a2).